### ISSUE TITLE
DeprecationWarning: the imp module is deprecated in favour of importlib

### ISSUE THREAD

**Issue by 1fish2** [viblo/pymunk#184]
Running `pytest` on our project in Python 3.8.3 prints a `DeprecationWarning` from `pymunk`:

```
========================================================================== test session starts ==========================================================================
platform darwin -- Python 3.8.3, pytest-5.4.3, py-1.9.0, pluggy-0.13.1
.
.
.
=========================================================================== warnings summary ============================================================================
/usr/local/var/pyenv/versions/3.8.3/envs/wcEcoli3/lib/python3.8/site-packages/pymunk/_libload.py:5
  /usr/local/var/pyenv/versions/3.8.3/envs/wcEcoli3/lib/python3.8/site-packages/pymunk/_libload.py:5: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
    import sys, imp, os

-- Docs: https://docs.pytest.org/en/latest/warnings.html
=============================================================== 166 passed, 2 skipped, 1 warning in 9.66s ===============================================================
```

Apparently pymunk should move from `imp` to `importlib`.

**Comment by viblo** [comment]
Thanks for the report!

I'm in the process of upgrading how pymunk uses the underlying c library
(which is where the deprecated logic is used). I will make sure to fix it
as part of the bigger change.

**Comment by viblo** [comment]
I have just released Pymunk 6.0.0. This version does not use imp anymore so should work without warnings. Feel free to try it out! (Note however that this version contains several api breaking changes, so if you have existing code you will very likely need to do some adjustments)
